.github/CODEOWNERS

```diff
@@ -881,6 +885,7 @@
 /eng/spotbugs-aggregate-report/                      @srnagar @JonathanGiles
 /eng/versioning/                                     @alzimmermsft @samvaity @g2vinay @JimSuplizio
 /eng/versioning/external_dependencies.txt            @alzimmermsft @samvaity @g2vinay @jonathangiles @rujche @netyyyy @saragluna @moarychan
+/.github/CODEOWNERS                                  @alzimmermsft @srnagar @Azure/azure-sdk-eng
```

Comment:
please add back the larger list of owners here since this file gets changed so often.